pkg/k8s/controllers/requeue.go

```diff
@@ -0,0 +1,41 @@
+package controllers
```

Comment:
could you not use an exponential backoff algorithm here - see the retry package that is either in services or this repo, can't remember

Reply:
added, thanks!